letsencrypt/defaults/main.yml

```diff
@@ -14,6 +14,10 @@ letsencrypt_certs: []
 letsencrypt_flags: ""
 letsencrypt_run: true
 
+# needs to be the same name as the variable in the nginx role and defined here as well
+# so that those two roles can be run independently
+nginx_enable_custom_domains: false
```

Comment:
This is still coupled. Call this `letsencrypt_enable_default_server` and couple it to `nginx_enable_custom_domains` in the server vars or playbook rather than in the role.

Reply:
Ah, ok, I see what you mean now. Will change it.